coriolis/tests/scheduler/test_scheduler_utils.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2023 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+
+from unittest import mock
+
+from coriolis import constants
+from coriolis import exception
+from coriolis.scheduler import scheduler_utils
+from coriolis.tests import test_base
+
+
+class CoriolisTestException(Exception):
+    pass
+
+
+class SchedulerUtilsTestCase(test_base.CoriolisBaseTestCase):
+    """"Test suite for the Coriolis scheduler utils package."""
+
+    def setUp(self):
+        super(SchedulerUtilsTestCase, self).setUp()
+        self.scheduler_client = mock.MagicMock()
+        self.rpc_client_class = mock.MagicMock()
+        self.service = mock.MagicMock()
+        self.ctxt = mock.MagicMock()
+
+    def test_get_rpc_client_for_service(self):
+        scheduler_utils.RPC_TOPIC_TO_CLIENT_CLASS_MAP[
```

Comment:
I don't like directly changing the dict. For safety's sake, can you please use `mock.patch.dict` here? 
https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch.dict

Reply:
Also, what if the service topic is not `constants.WORKER_MAIN_MESSAGING_TOPIC`? Please add a case for that too.